samples/matter/lock/README.rst

```diff
@@ -143,6 +143,26 @@ FEM support
 
 .. include:: /includes/sample_fem_support.txt
 
+Setting Thread radio output power
+=====================================
+
+.. matter_door_lock_sample_setting_thread_output_power_start
+
+In this example, the Thread radio output power is set to 0 dBm for nRF52 and nRF53 devices and to +10 dBm for nRF21 FEM device. 
+You can change the default output power value by adding the ``CONFIG_OPENTHREAD_DEFAULT_TX_POWER`` config to your project or using it as a build flag.
+The maximum possible output power is different for boards that support FEM.
+
+To set default Thread output power add the ``-DCONFIG_OPENTHREAD_DEFAULT_TX_POWER=X`` build flag to your build command, where ``X`` is the value from range <-20, 5> dBm.
```

Comment:
I would skip the part of providing the KConfig config using the command line (so all from 155 to 162).